### PR TITLE
Realistic Names - Add strings for Khaki MX variants

### PR DESCRIPTION
**Pull request by Salluci**
**When merged this pull request will:**
- Title.

### CODE REVIEW

File: addons/realisticnames/stringtable.xml [hunk 1]
```diff
@@ -1959,6 +1968,9 @@
             <Chinesesimp>MX轻型机枪 (黑色)</Chinesesimp>
             <Turkish>MX LSW (Siyah)</Turkish>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_MX_SW_Khaki_Name">
+            <English>MX LSW (Khaki)</English>
```
Comment (bux) [inline addons/realisticnames/stringtable.xml:1972]:
```suggestion
            <English>MX LSW (Khaki)</English>
            <German>MX LSW (Khaki)</German>
```

File: addons/realisticnames/stringtable.xml [hunk 2]
```diff
@@ -1925,6 +1931,9 @@
             <Chinesesimp>MX突击步枪 (3连装榴弹-黑色)</Chinesesimp>
             <Turkish>MX 3GL (Siyah)</Turkish>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_MX_GL_Khaki_Name">
+            <English>MX 3GL (Khaki)</English>
```
Comment (bux) [inline addons/realisticnames/stringtable.xml:1935]:
```suggestion
            <English>MX 3GL (Khaki)</English>
            <German>MX 3GL (Khaki)</German>
```

File: addons/realisticnames/stringtable.xml [hunk 3]
```diff
@@ -1891,6 +1894,9 @@
             <Chinesesimp>MXC卡宾步枪 (黑色)</Chinesesimp>
             <Turkish>MXC (Siyah)</Turkish>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_MXC_Khaki_Name">
+            <English>MXC (Khaki)</English>
```
Comment (bux) [inline addons/realisticnames/stringtable.xml:1898]:
```suggestion
            <English>MXC (Khaki)</English>
            <German>MXC (Khaki)</German>
```

File: addons/realisticnames/stringtable.xml [hunk 4]
```diff
@@ -1857,6 +1857,9 @@
             <Chinesesimp>MX突击步枪 (黑色)</Chinesesimp>
             <Turkish>MX (Siyah)</Turkish>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_MX_Khaki_Name">
+            <English>MX (Khaki)</English>
```
Comment (bux) [inline addons/realisticnames/stringtable.xml:1861]:
```suggestion
            <English>MX (Khaki)</English>
            <German>MX (Khaki)</German>
```

File: addons/realisticnames/stringtable.xml [hunk 5]
```diff
@@ -1857,6 +1857,10 @@
...
+            <German>MX (Khaki)</German>
```
Comment (PabstMirror) [inline addons/realisticnames/stringtable.xml:1862]:
```suggestion
            <German>MX (Khaki)</German>
            <Chinese>MX（卡其）</Chinese>
            <French>MX (kaki)</French>
            <Spanish>MX (caqui)</Spanish>
            <Italian>MX (kaki)</Italian>
            <Polish>MX (khaki)</Polish>
            <Russian>MX (хаки)</Russian>
            <Czech>MX (khaki)</Czech>
            <English>MX (Khaki)</English>
            <Portuguese>MX (cáqui)</Portuguese>
            <Korean>MX (카키)</Korean>
            <Chinesesimp>MX（卡其色）</Chinesesimp>
            <Japanese>MX（カーキ）</Japanese>
            <Turkish>MX (Haki)</Turkish>
```

File: addons/realisticnames/stringtable.xml [hunk 6]
```diff
@@ -1891,6 +1895,10 @@
             <Chinesesimp>MXC卡宾步枪 (黑色)</Chinesesimp>
             <Turkish>MXC (Siyah)</Turkish>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_MXC_Khaki_Name">
+            <English>MXC (Khaki)</English>
+            <German>MXC (Khaki)</German>
```
Comment (PabstMirror) [inline addons/realisticnames/stringtable.xml:1900]:
```suggestion
            <German>MXC (Khaki)</German>
            <Chinese>MXC（卡其）</Chinese>
            <French>MXC (kaki)</French>
            <Spanish>MXC (caqui)</Spanish>
            <Italian>MXC (kaki)</Italian>
            <Polish>MXC (khaki)</Polish>
            <Russian>MXC (хаки)</Russian>
            <Czech>MXC (khaki)</Czech>
            <Portuguese>MXC (cáqui)</Portuguese>
            <Korean>MXC (카키)</Korean>
            <Chinesesimp>MXC（卡其色）</Chinesesimp>
            <Japanese>MXC（カーキ）</Japanese>
            <Turkish>MXC (Haki)</Turkish>
```

File: addons/realisticnames/stringtable.xml [hunk 7]
```diff
@@ -1925,6 +1933,10 @@
             <Chinesesimp>MX突击步枪 (3连装榴弹-黑色)</Chinesesimp>
             <Turkish>MX 3GL (Siyah)</Turkish>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_MX_GL_Khaki_Name">
+            <English>MX 3GL (Khaki)</English>
+            <German>MX 3GL (Khaki)</German>
```
Comment (PabstMirror) [inline addons/realisticnames/stringtable.xml:1938]:
```suggestion
            <German>MX 3GL (Khaki)</German>
            <Chinese>MX 3GL（卡其）</Chinese>
            <French>MX 3GL (kaki)</French>
            <Spanish>MX 3GL (caqui)</Spanish>
            <Italian>MX 3GL (kaki)</Italian>
            <Polish>MX 3GL (khaki)</Polish>
            <Russian>MX 3GL (хаки)</Russian>
            <Czech>MX 3GL (khaki)</Czech>
            <Portuguese>MX 3GL (cáqui)</Portuguese>
            <Korean>MX 3GL (카키)</Korean>
            <Chinesesimp>MX 3GL（卡其色）</Chinesesimp>
            <Japanese>MX 3GL（カーキ）</Japanese>
            <Turkish>MX 3GL (Haki)</Turkish>
```

File: addons/realisticnames/stringtable.xml (hunk 6 above)
Comment (Salluci) [reply addons/realisticnames/stringtable.xml:1900]: Capitalization?

Comment (PabstMirror) [reply addons/realisticnames/stringtable.xml:1913]: not sure, this is just based on vanilla tables
```
        <Key ID="str_a3_cfgweapons_arifle_mx_khk_f0">
          <Original>MX 6.5 mm (Khaki)</Original>
          <Chinese>MX 6.5 mm（卡其）</Chinese>
          <French>MX 6,5 mm (kaki)</French>
          <Spanish>MX 6,5 mm (caqui)</Spanish>
          <Italian>MX 6,5 mm (kaki)</Italian>
          <Polish>MX 6,5 mm (khaki)</Polish>
          <Russian>MX 6,5 мм (хаки)</Russian>
          <German>MX 6,5 mm (Khaki)</German>
          <Czech>MX 6,5 mm (khaki)</Czech>
          <English>MX 6.5 mm (Khaki)</English>
          <Portuguese>MX 6,5 mm (cáqui)</Portuguese>
          <Korean>MX 6.5 mm (카키)</Korean>
          <Chinesesimp>MX 6.5 毫米（卡其色）</Chinesesimp>
          <Japanese>MX 6.5 mm（カーキ）</Japanese>
          <Turkish>MX 6.5 mm (Haki)</Turkish>
```

File: addons/realisticnames/stringtable.xml [hunk 8]
```diff
@@ -1993,6 +2009,10 @@
             <Chinesesimp>MXM精准步枪 (黑色)</Chinesesimp>
             <Turkish>MXM (Siyah)</Turkish>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_MXM_Khaki_Name">
+            <English>MXM (Khaki)</English>
+            <German>MXM (Khaki)</German>
```
Comment (PabstMirror) [inline addons/realisticnames/stringtable.xml:2051]:
```suggestion
            <German>MXM (Khaki)</German>
            <Chinese>MXM（卡其）</Chinese>
            <French>MXM (kaki)</French>
            <Spanish>MXM (caqui)</Spanish>
            <Italian>MXM (kaki)</Italian>
            <Polish>MXM (khaki)</Polish>
            <Russian>MXM (хаки)</Russian>
            <Czech>MXM (khaki)</Czech>
            <Portuguese>MXM (cáqui)</Portuguese>
            <Korean>MXM (카키)</Korean>
            <Chinesesimp>MXM（卡其色）</Chinesesimp>
            <Japanese>MXM（カーキ）</Japanese>
            <Turkish>MXM (Haki)</Turkish>
```

File: addons/realisticnames/stringtable.xml [hunk 9]
```diff
@@ -1959,6 +1971,10 @@
             <Chinesesimp>MX轻型机枪 (黑色)</Chinesesimp>
             <Turkish>MX LSW (Siyah)</Turkish>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_MX_SW_Khaki_Name">
+            <English>MX LSW (Khaki)</English>
+            <German>MX LSW (Khaki)</German>
```
Comment (PabstMirror) [inline addons/realisticnames/stringtable.xml:2013]:
```suggestion
            <German>MX LSW (Khaki)</German>
            <Chinese>MX LSW（卡其）</Chinese>
            <French>MX LSW (kaki)</French>
            <Spanish>MX LSW (caqui)</Spanish>
            <Italian>MX LSW (kaki)</Italian>
            <Polish>MX LSW (khaki)</Polish>
            <Russian>MX LSW (хаки)</Russian>
            <Czech>MX LSW (khaki)</Czech>
            <Portuguese>MX LSW (cáqui)</Portuguese>
            <Korean>MX LSW (카키)</Korean>
            <Chinesesimp>MX LSW（卡其色）</Chinesesimp>
            <Japanese>MX LSW（カーキ）</Japanese>
            <Turkish>MX LSW (Haki)</Turkish>
```

File: addons/realisticnames/stringtable.xml [hunk 10]
```diff
@@ -1857,6 +1857,23 @@
...
+            <English>MX (Khaki)</English>
+            <German>MX (Khaki)</German>
+            <Chinese>MX（卡其）</Chinese>
+            <French>MX (kaki)</French>
+            <Spanish>MX (caqui)</Spanish>
+            <Italian>MX (kaki)</Italian>
+            <Polish>MX (khaki)</Polish>
+            <Russian>MX (хаки)</Russian>
+            <Czech>MX (khaki)</Czech>
+            <English>MX (Khaki)</English>
```
Comment (PabstMirror) [inline addons/realisticnames/stringtable.xml:1870]:
```suggestion
```
opps

**Comment by Salluci** [comment]
Note to self, if developing a videogame, just use Green instead of "Khaki"

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**